### PR TITLE
add `use-test` rule (fixes #39)

### LINKED ISSUE #39: `use-test` rule
Rule proposal: Enforce that AVA gets imported with `test` as a variable name.

### Bad
```js
var foo = require('ava');
let foo = require('ava');
const foo = require('ava');
import foo from 'ava';
```

### Good
```js
var test = require('ava');
let test = require('ava');
const test = require('ava');
import test from 'ava';

var test = require('foo');
import test from 'foo';
```

I think this should be a recommended rule at error level, especially considering that all rules assume that AVA is named `test`. What do you think?

### PR DESCRIPTION
**Pull request by jfmengels**
adds `use-test` rule (fixes #39) :)

### CODE REVIEW

File: docs/rules/use-test.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Ensure that AVA gets imported with `test` as the variable name
```
Comment (sindresorhus) [inline docs/rules/use-test.md]: > Ensure that AVA is imported with `test` as the variable name

File: rules/use-test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+module.exports = function (context) {
+	return {
+		ImportDeclaration: function (node) {
+			if (node.source.value === 'ava' && node.specifiers[0].local.name !== 'test') {
+				report(context, node);
+			}
+		},
+
```
Comment (sindresorhus) [inline rules/use-test.js]: No empty line here

File: rules/use-test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	arguments: [
+		{
+			type: 'Literal',
+			value: 'ava'
+		}
+	]
+};
+
+function report(context, node) {
+	context.report(node, 'AVA should be imported as test.');
```
Comment (sindresorhus) [inline rules/use-test.js]:
```js
context.report(node, 'AVA should be imported as `test`.');
```

**Comment by jfmengels** [comment]
:+1:. Updated

**Comment by sindresorhus** [comment]
Woot! :dancer: 

tumblr_mqmk6j8kkz1qdlh1io1_250

**Pull request merged**